_notes/why_must_i_learn_this.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Why Must I Learn This?
+---
+
+# “Why” matters
+
+“Why must I learn this?"
+
+I always ask this question as a kid. 
```

Comment:
should this start with "I always asked"?

i wonder if it should be in the past tense

Soon I stopped asking why

Soon you/we stop asking why